docs/user-manual/modules/ROOT/pages/camel-4x-upgrade-guide-4_8.adoc

```diff
@@ -60,6 +60,13 @@ The chat-with-tools feature was deprecated. Use the new `camel-langchain4j-tool`
 Continuing the multi-release tests cleanups, on this one, restricted methods from the `CamelTestSupport` class
 have been marked as final and cannot be extended.
 
+=== camel-debezium
```

Comment:
This is for 4.8.2 - so some of the older guides how they have specific notes for a patch release.

Reply:
Do you know an example?